assets/blocks/course-outline/module-block/module-block.scss

```diff
@@ -68,10 +68,8 @@ $block: '.wp-block-sensei-lms-course-outline-module';
 		display: block;
 		border: none;
 		height: 4px;
-		.wp-block-sensei-lms-course-outline.is-style-default & {
```

Comment:
@yscik I found that it was introduced in this commit: https://github.com/Automattic/sensei/commit/082bccef9b30c65dfac00cee46ad42256ace02e3

But it's without the lateral spacings in the frontend, so I removed it. Do you remember why we needed specificity only for the lateral spacings?

Reply:
This added spacing for the Minimal module style when the border was on the outline, I think. So it can go 👍